install/kubernetes/cilium/values.yaml

```diff
@@ -127,7 +127,8 @@ extraEnv: {}
 extraInitContainers: []
 
 # -- Additional agent hostPath mounts.
-extraHostPathMounts: []
+extraHostPathMounts:
+  []
```

Comment:
This doesn't match the style in which this file is currently written...

Reply:
Indeed, was there a particular tool you used to format them this way? I don't think that the previous developers working on this file were aware of any such tool.

Reply:
> Indeed, was there a particular tool you used to format them this way? I don't think that the previous developers working on this file were aware of any such tool.

https://github.com/redhat-developer/yaml-language-server

Yes. my vscode editor will automatically format it